ansible_collections/arista/avd/roles/eos_designs/python_modules/underlay/utils.py

```diff
@@ -197,9 +205,8 @@ def _get_l3_interface_cfg(self, l3_interface: dict) -> dict | None:
         if ip_address == "dhcp" and l3_interface.get("dhcp_accept_default_route", True):
             interface["dhcp_client_accept_default_route"] = True
 
-        # TODO: enable flow tracking once toggle is in place
-        # if self.shared_utils.is_cv_pathfinder_router:
-        #    interface["flow_tracker"] = {"hardware": self.shared_utils.wan_flow_tracker_name}
+        if (flow_tracker := self.shared_utils.get_flow_tracker(l3_interface, "node_l3_interfaces", f"node.l3_interfaces[{interface_name}]")) is not None:
```

Comment:
Please move the `interface` dict above, since we strip None value below in `strip_empties_from_dict`.